src/redux/rootReducer.js

```diff
@@ -0,0 +1,7 @@
+import { combineReducers } from "redux";
+
+const rootReducer = combineReducers({
+// Set up your reduce here
+});
```

Comment:
Well done but I would like to have some simple reducer, just to test the setup functionality.